data/soapvalidator/Auth/JWT/JWT-ZCS-2478.xml

```diff
@@ -150,8 +150,8 @@
 
 	</t:test_case>
 
-	<t:test_case testcaseid="zcs_2478SendMsg_02" type="bhr"
-		bugsids="zcs-2478">
+	<t:test_case testcaseid="zcs_2478SendMsg_02" type="deprecated"
```

Comment:
Instead of deprecated, do we want to introduce new label for non-public server?

Reply:
Yes test case is not permanently deprecated

Reply:
"bhr" = "bhr-tbd"

Reply:
Thank you Jitesh, Changed group type to tbd-bhr